Sources/Orbit/Components/Timeline.swift

```diff
@@ -15,19 +15,19 @@ public struct Timeline<Content: View>: View {
             VStack(alignment: .leading, spacing: Self.spacing) {
                 content
             }
-            .backgroundPreferenceValue(TimelineStep.PreferenceKey.self) { preferences in
+            .backgroundPreferenceValue(TimelineStepPreferenceKey.self) { preferences in
                 GeometryReader { geometry in
                     VStack(spacing: 0) {
                         ForEach(preferences.indices.dropFirst(), id: \.self) { index in
                             preferences[index].style.color
```

Comment:
Somewhere here the line should be correctly positioned for dynamic type

<img width="351" alt="image" src="https://user-images.githubusercontent.com/35844477/180400221-9f03d09f-cff9-4e49-be24-ec6ac2ac591a.png">

Reply:
Created new issue #241 for that. It's easy to align it horizontally but not vertically